inc/Engine/CDN/CDN.php

```diff
@@ -207,7 +205,7 @@ public function get_cdn_urls( $zones = 'all' ) {
 	 * @return string
 	 */
 	private function get_base_url() {
-		return '//' . wp_parse_url( get_option( 'home' ), PHP_URL_HOST );
+		return '//' . wp_parse_url( home_url(), PHP_URL_HOST );
```

Comment:
We are getting `home_url()` 2x in this class. That means it'll go through the filtering 2x. To make this more performant, let's move it to a private method and add a property. If the property is not populated, them get `home_url()`; else, return the property.

Reply:
https://github.com/wp-media/wp-rocket/pull/2589/commits/7c2ec6c8f0c776c7b86cf3a17d81daf192df446e